1.  tropomyosin
2.  α-actinin
3.  ketohexokinase
4. fibroblast growth factor receptor 2
5. **PKM**
	1. PKM1 is present in most differentiated cells
	2. PKM2 is present in most cell lines and proliferating cells
	3. U-118MG and A-172 glioblastoma cell lines still express PKM1 (maybe a good choice for library experiments?)
		1. Also SK-N-BE which splices the RNA both ways but seems to only express PKM1 anyway
	2. proliferating mouse C2C12 myoblasts induced to terminally differentiate into myotubes should switch isoform expression
		1. May allow measurement of how variants affect regulation during differentiation
6. SCN genes
	1. 6/10 voltage gated sodium channels
7. Glutamate receptor subunits 1-4
8. Snap-25
	1. Neuron exocytosis machinery
9. CACNA1C
10. SLC25A3
	1. mitochondrial phosphate carrier 
11. SLC39A14
	1. zinc transporter
12. **CUX1**
	1. Cut-like homeobox 1
	2. has 4 clusters (and an alt promoter)
		1. the exon 3 and 4 clusters are complicated and interleaved
		2. can pick 3a and 4a or 1 of 3b-e and 4b or one of 4a or 4b with no 3. but other combos are not possible
		3. Might make an interesting system
	3. Seems able to backsplice to make Circ-RNAs
	4. associated with cancer but in confusing ways
13. **ATE1**
	1. Cancer and developmental relevance
	2. Has MXEs that appear to be controlled by 2 sets of competing secondary structures
	3. Since the relevant sites are already known, and ASOs exist, we could try to directly drill into how splicing regulation by secondary structure works
14. Found the website https://kassiopeia.motorprotein.de/kassiopeia_proteins?window_id=944162377
	1. Has a filterable dataset of genes with predicted MXEs in them 
	2. Website is slow and hard to navigate, but there seem to be lots of human proteins with predicted complicated MXE structures
15. **CD55**
	1. Has a cluster of 5 predicted MXEs
	2. Is important for a variety of diseases
		1. Cancer
		2. Thrombosis 
		3. Others
	3. Can be expressed on HeLa cells
	4. Has predicted docker selector sequences
16. DISC1
	1. Has interleaved MXEs 
	2. One cluster has 8 MXEs
	3. Important for neuronal development
17. TPM1
	1. exons 2 and 3 are a well known model of Mutually exclusive splicing
		1. Steric hindrance mechanism
	2. Also shows distant coordinated splicing. both co-inclusion and MXE
		1. [[Zhu_et_al-2021-Single-molecule_full-length_transcript_isoform_sequencing_reveals_disease-associated_RNA_isoforms_in_cardiomyocytes|Zhu et al 2021]] 
18. MYL1
	1. Crucial muscle gene
	2. Has many examples of co-regulated exons
	3.  [[Zhu_et_al-2021-Single-molecule_full-length_transcript_isoform_sequencing_reveals_disease-associated_RNA_isoforms_in_cardiomyocytes|Zhu et al 2021]] 
19. Fibronectin
	1. Has corrolated splicing between two exons
	2. Correlation seems to relate to transcription speed
	3. Fededa, J. P. _et al._ A polar mechanism coordinates different regions of alternative splicing within a single gene. _Mol. Cell_ **19**, 393–404 (2005)
20. MAPT
	1. Encodes Tau -> relevent to Taupathies
	2. Has a pair of exons which are non-adjacent but seem not to appear together
		1. can both be missing
	3. [[Tilgner_et_al-2018-Microfluidic_isoform_sequencing_shows_widespread_splicing_coordination_in_the_human_transcriptome|tilgner et al 2018]]
21. 