%!TEX output_directory = build
%!TEX copy_output_on_build = true

\documentclass{article}

\usepackage[
backend=biber,
style=numeric-comp,
]{biblatex}
\addbibresource{./paperpile.bib} %Imports bibliography file

\usepackage{soul}

\usepackage[letterpaper,
            bindingoffset=0.2in,
            left=1in,
            right=1in,
            top=1in,
            bottom=1in,
            footskip=.25in]{geometry}

\usepackage{titlesec} 
\titleformat{\section}[runin]
  {\normalfont\normalsize\bfseries}{}{0pt}{}
\titlespacing{\section}{0pt}{-\parskip}{1mm}

\renewcommand{\thesubsection}{\arabic{subsection}}
\titleformat{\subsection}[runin]
  {\normalfont\normalsize\bfseries}{Aim \thesubsection: }{0pt}{}
\titlespacing{\subsection}{0pt}{-\parskip}{1mm}

\titleformat{\subsubsection}[runin]
  {\normalfont\normalsize\itshape}{Sub-aim \thesubsubsection: }{0pt}{\ul}
\titlespacing{\subsubsection}{0pt}{-\parskip}{1mm}

\begin{document}
\section*{Specific Aims:}
%
\section*{Rationale: } 
Splicing is a process where pre-mRNAs are processed to remove introns and stitch together exons. 
This process offers the potential to introduce tremendous diversity into the proteome by changing which sequences from the pre-mRNA are included as exons.
Accordingly, more than 90\% of human genes are alternatively spliced in different conditions, such as at different developmental stages or across cell types.\cite{Wang2008-ej} 
Errors in this process have been implicated in a wide variety of human diseases including familial dysautonomia, early onset Parkinson disease, and cancer.\cite{Scotti2015-yp} 
Further recent efforts have produced drugs that modulate splicing outcomes to produce improvements in Spinal muscular Atrophy, Huntington's Disease, Duchenne muscular dystrophy, and cancer.\cite{Neil2022-vf}
Work to understand the mechanisms of splicing stands to benefit not only our understandings of development and differentiation but also our ability to target these processes with therapeutics.
Much progress has been made using massively parallel splicing assays (MPSAs) to probe the mechanisms of splicing.\cite{Ke2018-af, Julien2016-wa, Adamson2018-va, Soemedi2017-pz, Cortes-Lopez2022-gy, Schirman2021-ss, Mikl2019-ng, Braun2018-mb, Soucek2019-iq, Baeza-Centurion2020-tn, Cheung2019-ah, Baeza-Centurion2019-hz, Rosenberg2015-zs, Wong2018-vq} 
These efforts have largely focused on understanding how the sequence characteristics of the pre-RNA dictate the inclusion or exclusion of a potential exon and have made great strides.
However, while most alternative splicing events are believed to be local affecting an individual intron or exon, recent work has begun to show that correlated splicing decisions are far more common than was previously understood.\cite{Zhu2021-fs, Tilgner2015-sb, Hatje2017-oj} 
A particularly interesting case of correlated splicing is mutually exclusive exons (MXEs). 
MXEs are clusters of exons that are spliced such that every isoform includes exactly one exon from the cluster. 
While in other species clusters of more that 2 MXEs are well known, examples in humans have only recently been detected and have never been extensively characterized.\cite{Jin2018-tq, Hatje2017-oj} 
A recent analysis of human RNA-seq data identified 629 MXE clusters, of of which less than 25\% had an identifiable mechanism for enforcing mutual exclusivity.\cite{Hatje2017-oj} 
\textbf{I propose to develop a massively parallel splicing assay for characterizing mechanisms of mutual exclusivity in splicing and apply it to investigating new mechanisms of mutual exclusivity in human splicing. I will focus these efforts on Pyruvate Kinase M (PKM) and CD55.} 
\\
\subsection{Develop an MPSA protocol capable of handling more diverse isoforms.}
Current MPSAs utilize Illumina technologies to read out the population of the various isoforms they detect.\cite{Ke2018-af, Julien2016-wa, Adamson2018-va, Soemedi2017-pz, Cortes-Lopez2022-gy, Schirman2021-ss, Mikl2019-ng, Braun2018-mb, Soucek2019-iq, Baeza-Centurion2020-tn, Cheung2019-ah, Baeza-Centurion2019-hz, Rosenberg2015-zs, Wong2018-vq} 
However, this decision puts strict length limits on reads that can be obtained and therefore restricts the isoforms that can be reliably detected.
As mutations in MXE clusters can cause the production of isoforms that include part or all of the introns in addition to just inappropriate inclusion or exclusion of exons,\cite{Wang2012-dr} longer reads are needed to fully characterize mutated MXE clusters with their native introns. 
Further, a long read MPSA opens the door to studies of more complicated correlated splicing events like large MXE clusters\cite{Hatje2017-oj} or long distance splicing correlations.\cite{Zhu2021-fs, Tilgner2015-sb, Tilgner2018-jo}
%
\subsubsection{Create an MPSA protocol utilizing nanopore sequencing technology.} \label{aim:nanopore_screen_dev}
%
\subsubsection{Develop an analysis pipeline for quantifying isoform abundances from MPSA data.} \label{aim:pipeline_QC}
%
\subsection{Characterize the mechanism for mutual exclusivity in Pyruvate Kinase M (PKM) exons 9 and 10.}
PKM is a central metabolic gene that controls the rate limiting step in glycolysis with two primary isoforms defined by the inclusion of either exon 9 or exon 10. 
The conversion between these two isoforms is a primary driver of the Warburg effect and is important in a wide variety of cancers.\Cite{Christofk2008-bu,Ma2022-dt} 
However, while much is known about the regulation of PKM splicing, the mechanism for enforcing exclusivity between exons 9 and 10 is still unknown.
%
\subsubsection{Develop a screen to detect deviations from mutual exclusivity in splicing.} \label{aim:screen_dev}
We will utilize PKM minigene constructs designed by the Krainer lab for studying PKM splicing \cite{Wang2012-dr} and adapt MPSA techniques honed by the Kinney lab \cite{Wong2018-vq,Ishigami2022-bf} to develop a massively parallel assay for detecting mutual exclusivity or an MX-MPSA.
In order to create the MX-MPSA, we will adjust the sequencing strategy to utilize next generation sequencing platforms capable of sequencing the entire length of the minigene in order to observe the widest array of possible splicing outcomes.
We will then measure the isoform distribution of each mutant as a function of varying the inclusion ratios of wild-type MXEs using splice modifying drug titrations.
We will trial using modulating splice ratios in wild type genes with ASOs\cite{Wang2012-ea,Ma2022-dt} or by engineering in risdiplam sensitive elements and using drug titration MPSA techniques\cite{Ishigami2022-bf}. 
Any deviations from anti-correlation between MXEs will indicate that that mutant disrupted an element important for the maintenance of mutual exclusivity(MMX). We will validate our analysis pipeline on simulated data, and validate our experimental pipeline against gold standard rt-qPCR and rt-PCR band analysis experiments designed PKM mutants.
% 
\subsubsection{Identify sequence elements necessary for mutual exclusivity in PKM splicing.} \label{aim:PKM_motif_finding}
To identify the MMX elements for exons 9 and 10 in PKM, we propose to perform a mutant screen using the MX-MPSA technique we developed in sub-aim \ref{aim:screen_dev}. 
Previous work has shown that MMX elements for PKM are located internal to exons 9 and 10,\cite{Wang2012-dr} so we will utilize saturation mutagenesis of these exons to identify these elements.
We will couple these experiments with broader random mutagenesis strategies to map MMX elements that may also reside in the introns. 
Identified regions will be mapped at higher resolution with deep mutagenesis and our MX-MPSA technique. If the MX-MPSA proves challenging to develop, we will pursue the same approach using lower throughput methods and rt-qPCR or rt-PCR band analysis approaches.
%
\subsubsection{Mechanistically characterize sequence elements enforcing mutual exclusivity.} \label{aim:PKM_mechanistic}
We will follow up identification of MMX elements in sub-aim \ref{aim:PKM_motif_finding} with mechanistic studies of the sequence elements that enforce mutual exclusivity. 
In many arthropods and the human ATE1 gene \cite{Graveley2005-oi, Jin2018-tq, Kalinina2021-jt} MMXs are RNA secondary structural elements. 
We will use mutant cycle analysis to show dependence on structure for any identified secondary structural elements. 
We will also use RNA based pull downs as well as motif mutants coupled with RNAi knockdown and over-expression of RNA binding proteins (RBPs) to demonstrate the importance identified RBP binding motifs.
This data will be used to inform a mechanistic model of PKM MXE splicing.







\printbibliography

\end{document}










