1. How are MXEs kept MX
2. How do intron and exon size cause intron def, exon def, and back splicing
3. Do helicases enable a kinetic proofreading model of psuedo cooperatively
4. How does 5' u1 binding energy affect 5' site choice and is there position bias?
	1. [[Mikl_et_al.-2019-cell_variability_with_designed_sequence_libraries|Mikl et al. 2019]] showed a bias towards the more 5' of pairs of 5'-ss seqs 
		1. They see that the strong site dominates but when they are equal the 5'-ss more 5' is prefered
		2. Amount of preference for the 5' seq seems to depend on seq context
		3. Could be fun to try systematically adjusting 5'-ss strength of the the two sites and the offset to see how much bias I get
5. When variants are miss spliced multiple ways, exon skipping vs intron retention, vs alternate site choice. what processes are competing
6. Does RECTAS really act through a 5' site mechanism
7. Do epi-transcriptome marks change splicing outcomes??
8. Can we measure RNA binding protein affinities for splicable sequences
	1. Maybe using arrays a la [[2023_02_15_QB_seminar]]
9. 