%!TEX output_directory = build
%!TEX copy_output_on_build = true

\documentclass{article}

\raggedright

\usepackage[style=authoryear, 
            backend=biber,
            url=false,
            eprint=false, 
            dashed=false,
            maxnames=999]{biblatex}
\addbibresource{./mypapers.bib} %Imports bibliography file

\newcounter{namesnotimportant}
\newtoggle{ellipsis}

\makeatletter
\newbibmacro*{name:etal:delim}[1]{%
  \ifnumgreater{\value{listcount}}{\value{liststart}}
    {\ifboolexpr{
       test {\ifnumless{\value{listcount}}{\value{liststop}}}
       or
       test \ifmorenames
       or test {\ifnumcomp{\value{namesnotimportant}}{>}{0}}
     }
       {\printdelim{multinamedelim}}
       {\lbx@finalnamedelim{#1}}}
    {}}
\makeatother

\DeclareNameFormat{given-family-etal}{%
  \letbibmacro{name:delim}{name:etal:delim}%
  \ifnumcomp{\value{listcount}}{=}{1}
    {\setcounter{namesnotimportant}{0}%
     \global\toggletrue{ellipsis}}
    {}%
  \ifboolexpr{test {\ifnumcomp{\value{listcount}}{=}{1}}
              or test {\ifnumcomp{\value{listtotal}}{=}{2}}}
    {\ifgiveninits
      {\usebibmacro{name:given-family}
         {\namepartfamily}
         {\namepartgiveni}
         {\namepartprefix}
         {\namepartsuffix}}
      {\usebibmacro{name:given-family}
         {\namepartfamily}
         {\namepartgiven}
         {\namepartprefix}
         {\namepartsuffix}}}%
    {\ifboolexpr{test {\iffieldequalstr{hash}{d044631f3f9582c1e40e11481a6d06e9}}
            or 
            test {\iffieldequalstr{hash}{d36f79db286d144cbc39cc5559a702d3}}
            or 
            test {\iffieldequalstr{hash}{02913108490bef3778a320768a68b0d9}}}%% <----- put the correct hash here
      {\global\toggletrue{ellipsis}%
       \ifgiveninits
        {\usebibmacro{name:given-family}
           {\namepartfamily}
           {\namepartgiveni}
           {\namepartprefix}
           {\namepartsuffix}}
        {\usebibmacro{name:given-family}
           {\namepartfamily}
           {\namepartgiven}
           {\namepartprefix}
           {\namepartsuffix}}}%
      {\stepcounter{namesnotimportant}%
       \iftoggle{ellipsis}
         {\addcomma\space\textellipsis\global\togglefalse{ellipsis}\isdot}
         {}}}%
  \ifboolexpr{
    test {\ifnumequal{\value{listcount}}{\value{liststop}}}
    and
    (test \ifmorenames
     or test {\ifnumcomp{\value{namesnotimportant}}{>}{0}})
  }
    {\andothersdelim\bibstring{andothers}}
    {}}

\DeclareNameAlias{sortname}{given-family-etal}
\DeclareNameAlias{author}{given-family-etal}
\DeclareNameAlias{editor}{given-family-etal}
\DeclareNameAlias{translator}{given-family-etal}


\usepackage{soul}

\usepackage[letterpaper,
            bindingoffset=0.2in,
            left=1in,
            right=1in,
            top=1in,
            bottom=1in,
            footskip=.25in]{geometry}

\usepackage{titlesec} 
\titleformat{\section}[runin]
  {\normalfont\normalsize\bfseries}{}{0pt}{}
\titlespacing{\section}{0pt}{-\parskip}{1mm}

\renewcommand{\thesubsection}{\arabic{subsection}}
\titleformat{\subsection}[runin]
  {\normalfont\normalsize\bfseries}{Contribution \thesubsection: }{0pt}{}
\titlespacing{\subsection}{0pt}{-\parskip}{1mm}

\titleformat{\subsubsection}[runin]
  {\normalfont\normalsize\itshape}{\thesubsubsection: }{0pt}{\ul}
\titlespacing{\subsubsection}{0pt}{-\parskip}{1mm}

\begin{document}


\section*{Contributions to Science:}
\newrefsection
\subsection{Mechanisms and evolution of carbon pumps that drive CO$_2$ concentration:}
\subsubsection{Historical background}
Rubisco is the enzyme responsible for fixing $>$99.5\% of all the CO$_2$ that enters the biosphere each year and is essential for all plants, algae, and most autotrophic bacteria. 
However, Rubisco is inhibited by the presence of O$_2$, a problem in the the modern atmosphere, with its 20\% O$_2$ and only 0.04\% CO$_2$.
Many bacteria overcome this using an $\alpha$-carboxysome based CO$_2$ concentrating mechanism (CCM).
These systems start by pumping HCO$_{3}^{-}$ into the cytosol, this HCO$_{3}^{-}$ then enters a protein compartment where it is converted into CO$_2$ raising the concentration high enough to saturate rubisco and out-compete O$_2$.
While this system had been worked out over decades of experimental and theoretical work, the mechanism of HCO$_{3}^{-}$ pumping was unknown.
Further, the CCM appeared irreducibly complex so a better characterization of possible ancestral CCMs was critical to understanding how it could have evolved.
In my graduate studies, I set out to address these questions.
%
\subsubsection{Discovery of a new type of inorganic carbon (C$_i$) pump that drives CO$_2$ concentration in bacteria}
To identify the missing HCO$_{3}^{-}$ pumps,I performed an unbiased screen for CCM genes in the model chemotroph \textit{H. neapolitanus}.
I identified two putative transporter genes operons, then confirmed their activity with growth and carbon uptake assays in \textit{E. coli}. 
I showed that the data were consistent with transport occurring through energy coupled carbonic anhydrase (CA) activity not direct pumping.
These CAs concentrate C$_i$ by converting membrane permeable CO$_2$ into membrane impermeable HCO$_{3}^{-}$ trapping the C$_i$ in the cell.
Further similar operons in the human pathogens \textit{V. Cholera} and \textit{B. anthracis} had the same activity.
As the primary author on this work, I conceived and designed the experiments, performed the genetic screens, analyzed sequencing data, did cloning, performed the biochemistry experiments, and performed the reporter strain experiments.
This work was instrumental in ongoing study of the CCM including a successful effort to reconstitute a functional $\alpha$-carboxysome CCM in \textit{E. coli}, and work on CCM evolution that I will discuss in \ref{CCM_EVO}. 
Since this work was published, it was shown that a similar operon in \textit{S. aureus} has the same function and is essential for growth in air. 
Considering the activity of energy coupled CA pumps has helped understand the carbon isotope fractionation data in very old rock strata.
There has been interest in investigating the potential applications of DABs in engineering crop plants and autotrophic bio-fuel production hosts (like \textit{C. necator}) for increased yield.
%
\nocite{Desmarais2019-yc,Desmarais2018-ac,Desmarais2019-dc}
\printbibliography[heading=none]

% \leavevmode\newline
\leavevmode\pagebreak

\newrefsection
\subsection{Mechanisms and evolution of carbon pumps that drive CO$_2$ concentration:}\label{CCM_EVO}
\subsubsection{Historical background}
Rubisco is the enzyme responsible for fixing $>$99.5\% of all the CO$_2$ that enters the biosphere each year and is essential for all plants, algae, and most autotrophic bacteria. 
However, Rubisco is inhibited by the presence of O$_2$, a problem in the the modern atmosphere, with its 20\% O$_2$ and only 0.04\% CO$_2$.
Many bacteria overcome this using an $\alpha$-carboxysome based CO$_2$ concentrating mechanism (CCM).
These systems start by pumping HCO$_{3}^{-}$ into the cytosol, this HCO$_{3}^{-}$ then enters a protein compartment where it is converted into CO$_2$ raising the concentration high enough to saturate rubisco and out-compete O$_2$.
While this system had been worked out over decades of experimental and theoretical work, the mechanism of HCO$_{3}^{-}$ pumping was unknown.
Further, the CCM appeared irreducibly complex so a better characterization of possible ancestral CCMs was critical to understanding how it could have evolved.
In my graduate studies, I set out to address these questions.
%
\subsubsection{Characterization of potential evolutionary paths for developing carbon dioxide concentrating mechanisms}
The $\alpha$-carboxysome based CO$_2$ concentrating mechanism (CCM) required three major evolutionary steps, acquiring a CA, gaining a C$_i$ pump, and co-encapsulating CA and rubisco.
All of these components are essential in the modern CCM raising questions about how the system could have evolved.
However, the atmosphere when the CCM first evolved had more CO$_2$ and less O$_2$, so we hypothesized that evolutionary intermediates may have provided fitness benefits in prior atmospheres.
We compared the effect of CCM gene knockouts in \textit{H. neapolitanus} across different CO$_2$ concentrations.
We also compared the CO$_2$ dependent phenotypes of potential evolutionary intermediates in CCM dependent strains of \textit{E. coli} and \textit{C. necator} that we constructed.
Modeling suggested that at intermediate CO$_2$ concentrations and high growth rates the cell can become limited by both CO$_2$ and HCO$_{3}^{-}$ and possessing a CA and a pump becomes beneficial despite potential for a futile cycle.
This revealed a possible path to evolving a CCM by acquiring first either a pump or CA, then acquiring the other as CO$_2$ fell further, and finally evolving encapsulation as CO$_2$ concentrations approached modern levels.
My role in this project was to measure the effect of all CCM gene knockouts on the growth of \textit{H. neapolitanus} across a panel of intermediate CO$_2$ concentrations to establish which components of it's CCM are needed for growth at intermediate CO$_2$ concentrations.
This work provides insight into the evolution of the CCM and into possible life strategies of modern organisms living in high CO$_2$ environments.
These strategies might also be useful for improving the growth of industrial autotrophs at intermediate CO$_2$ concentrations.
Further, showing the functional expression of the DAB in \textit{C. necator} offers the potential for using the DAB to improve bio-plastics production.

%
\nocite{Flamholz2022-yo}
\printbibliography[heading=none]

% \leavevmode\newline
\leavevmode\pagebreak

% \newrefsection
\subsection{General epistasis protein fitness landscape mapping and design:}
\subsubsection{Historical background}
Mutational scanning maps a protein's fitness landscape by measuring the fitness of all single mutants.
This information is used for variant effects prediction and design.
However, mutational scanning experiments are difficult because they require production of all single mutants and they only provide information on fitness in the protein's local context limiting their utility for divergent proteins.
Being able to learn from data sets that are easier to generate and provide information over a wider area would be greatly beneficial.
Current efforts to learn from random mutagenesis have relied on large neural networks or linear methods.
However, linear methods miss nonlinearities in the data's true structure and neural networks cannot be inspected to gain insight into function.
General epistatic models capture nonlinear genotype-phenotype relationships without sacrificing interpretability but have not been applied to protein design.
Phylogentic data also provide insight into fitness landscapes but inspectable models that are able to leverage both phylogenetic and experimental data have been rare.
%
\subsubsection{Central finding}
We performed random mutagenesis on dihydrofolate reductase (DHFR) and measured enzyme activity in a massively parallel growth assay. 
I trained models including simple linear models, general epistatic models, and large neural nets with and without access to phylogenetic data. 
I showed that the general epistatic models had simmilar performance to the neural net on held out training data and out performed the phylogenetic data alone or with the linear model. 
I also used each of these models to design new DHFR variants while systematically varying both number of mutations and optimization strategy to evaluate how well each model is able to extrapolate.
I used a massively parallel growth assay to validate 12,000 designed proteins.
This work is currently awaiting sequencing results from the final experiment before submission.
%
\subsubsection{Influence/Application}
This work provides new methods for mapping the genotype-phenotype landscape of proteins and designing new variants.
It also provides simple directly inspectable models that produce performance on par with neural net approaches for some prediction and design tasks.
%
\subsubsection{My role}
I am the primary author on this work.
I designed and conceived of the experiments, produced mutant libraries, performed massively parallel growth assays, wrote analysis code, wrote model code, trained models, tested model performance, and evaluated optimized sequence behavior.
%
% \printbibliography[heading=none]

% \leavevmode\newline
\leavevmode\pagebreak


\newrefsection
\subsection{Development of new CRISPR tools:}
\subsubsection{Historical background}
Cas proteins have found uses ranging from knocking out genes, to controlling expression, to detecting nucleic acids.
Crafting new CRISPR tools relies on discovery of new natural proteins and combining already known proteins in new ways.
During my graduate career I helped with two main CRISPR tool projects, identifying CasX as a new RNA guided DNA nuclease and the development of Cas13/Csm6 RNA diagnostics.
%
\subsubsection{CasX genome editing}
RNA guided DNA nucleases launched the CRISPR field.
However, large nuclease size has hindered therapeutic applications.
CasX was discovered as a <1,000 amino acid RuvC containing protein in CRISPR loci, with little homology to known nucleases, capable of protecting bacteria from transformation.
We demonstrated guided cutting activity \textit{in vitro} as well as gene knockout and CRISPRi \textit{in vivo}.
We then solved a Cryo-EM structure of the complex with target DNA and identified two new domains.
We also detected a putative zinc binding motif and showed that casX purifies bound to zinc.
My role in this project was to use x-ray fluorescence spectroscopy to identify the zinc bound to the purified protein.
CasX provides a new modality for genome editing that is proving useful in a variety of applications and provides hope for aiding the treatment of a wide variety of diseases.
Scribe Therapeutics is pursuing casX based therapies.
%
\subsubsection{Csm6 boosted Cas13 RNA detection}
CRISPR diagnostics allow detection of nucleic acids without multiple liquid handling or incubation steps.
This makes them attractive for use in at-home or point-of-care diagnostics.
However, they were not sensitive enough to detect SARS-COV2 in patient samples without pre-amplification of target sequences.
Class III CRISPR systems include a cyclic-oligo-A activated nuclease, Csm6, to strengthen the immune response.
We hypothesized linking cas13 detection of viral RNAs to csm6 activation would amplify the signal and allow SARS-COV2 detection in clinical samples.
We designed collateral substrates for cas13 that would activate Csm6.
We used kinetic modeling and spike in assays to show that activator self cleavage was limiting sensitivity.
Using an uncleavable activator allowed detection of SARS-COV2 in clinical samples.
My role in this work was kinetic modeling of potential reaction schemes to predict behavior and writing the data analysis and statistics pipelines.
This work has improved time to detection and sensitivity in CRISPR diagnostics and allow detection of clinically relevant nucleic acids.
%

\nocite{Liu2021-pu,Liu2019-nk}
\printbibliography[heading=none]

% \leavevmode\newline
\leavevmode\pagebreak


\newrefsection
\subsection{Driving carbon flux toward chemical production by engineering glucose uptake during nitrogen starvation:}
\subsubsection{Historical background}
Using microbial hosts to produce chemicals offers the potential to produce a variety of compounds from renewable feed-stock.
However, production hosts frequently loose production efficiency as natural selection drives evolution towards redirecting carbon and energy flux towards growth not production.
This can be overcome by coupling production of the desired chemical to cell fitness, but in many cases this is not possible.
An alternative strategy is growth decoupling, in which production hosts are grown up, then growth is stopped and all metabolic flux is directed towards production.
However, when growth is stopped, many chassis organisms including \textit{E. coli} slow and eventually halt their metabolism stopping production.
A general strategy for enhancing metabolic rate during growth decoupling would dramatically improve prospects for engineered chemical production in biological hosts.
%
\subsubsection{Central finding}
We found that by over-expressing PstI we were able to increase glucose uptake after growth was stopped by nitrogen limitation. 
However we did not find this increased glucose consumption increased yield significantly, and it is likely additional work will be needed to direct this increased flux towards chemical production.
%
\subsubsection{Influence/Application}
This work provides another tool that metabolic engineers can use to optimize the production of their compound of interest.
%
\subsubsection{My role}
My role in this project was to perform growth and chemical production assays with modified strains and to prepare samples for mass spectrometry.
%
\nocite{Chubukov2017-uu}
\printbibliography[heading=none]

% \leavevmode\newline
\leavevmode\pagebreak


\newrefsection
\subsection{Discovery of a new type of carbon pump that drives CO$_2$ concentration in bacteria:}
\subsubsection{Historical background}
Rubisco is the enzyme responsible for fixing the vast majority of all the CO$_2$ that enters the biosphere each year and is essential for all plants, algae, and most autotrophic bacteria. 
However, Rubisco evolved before the great oxygenation event and is competitively inhibited by the presence of O$_2$.
This is a problem that all autotrophs that use rubisco must overcome to survive in the the modern atmosphere, with its 20\% O$_2$ and only 0.04\% CO$_2$.
Many autotrophic bacteria including a wide range of chemolithoautotrophs and most cyanobacteria overcome this issue using an $\alpha$-carboxysome based CO$_2$ concentrating mechanism (CCM).
These CCMs work by creating a compartment inside the bacteria where the concentration of CO$_2$ is raised high enough to saturate rubisco with CO$_2$ and out-compete O$_2$.
While our theoretical understanding and previous experimental work suggested that an inorganic carbon (C$_i$) transporter was absolutely required for the functioning of the CCM, no such transporter was known in the model chemolithoautotroph \textit{H. neapolitanus}.
I identified putative transporters, .
%
\subsubsection{Central finding}
I performed a genome-wide genetic screen for CCM components in \textit{H. neapolitanus}. 
I identified the essential components of the CCM in \textit{H. neapolitanus}.
I cloned putative C$_i$ transporters, called DABs, then confirmed their activity with reporter strain and C$_i$ uptake assays in \textit{E. coli}. 
I showed that data was consistent with these transporters acting not as direct transporters but as a new family of energy coupled carbonic anhydrase (CA) enzymes.
These CAs concentrate C$_i$ by converting membrane permeable CO$_2$ into membrane impermeable HCO$_{3}^{-}$ causing a net flow of C$_i$ into the cell.
I further showed that similar operons in the human pathogens \textit{V. Cholera} and \textit{B. anthracis} had the same function.
%
\subsubsection{Influence/Application}
This work was instrumental in ongoing study of the CCM including a successful effort to reconstitute a functional $\alpha$-carboxysome CCM in \textit{E. coli}, and work on CCM evolution that I will discuss in contribution \ref{CCM_evo}. 
Since this work was published, it was shown that a similar operon in \textit{S. aureus} has the same function and is essential for growth in atmospheric CO$_2$ concentrations. 
Considering the activity of energy coupled CA transporters has been shown to be necessary to understand the carbon isotope fractionation data in very old rock strata.
There has been interest in investigating the potential applications of DABs in engineering crop plants and autotrophic bio-fuel production hosts (like \textit{C. necator}) for increased yield.
%
\subsubsection{My role}
I was the primary author on this work.
I conceived and designed the experiments, performed the genetic screen, analyzed the sequencing data, did cloning, performed the biochemistry experiments, and performed the reporter strain experiments.
%
\nocite{Desmarais2019-yc,Desmarais2018-ac,Desmarais2019-dc}
\printbibliography[heading=none]

% \leavevmode\newline
\leavevmode\pagebreak


\newrefsection
\subsection{Characterization of potential evolutionary paths for developing carbon dioxide concentrating mechanisms:} \label{CCM_evo}
\subsubsection{Historical background}
The $\alpha$-carboxysome based CO$_2$ concentrating mechanism (CCM) required several major evolutionary steps to evolve.
These were acquiring uncoupled CA activity, gaining C$_i$ transport, and encapsulating CA and rubisco in an $\alpha$-carboxysome.
However, all of these components are need for the effect of the CCM and removing even one of these components is lethal in modern autotrophs.
Since there is no apparent fitness benefit for a partial system, it is not clear how the system could have evolved.
Geochemical evidence suggests that the atmosphere was very different when the CCM first evolved, with much higher levels of CO$_2$ and much lower levels of O$_2$ and a mixture of biological and geochemical processes has slowly changed the atmospheric composition to the current mixture.
This information lead us to the hypothesis that evolutionary intermediates of the CCM may have provided fitness benefits at intermediate atmospheric compositions.
%
\subsubsection{Central finding}
We compared the effect of CCM gene knockouts in \textit{H. neapolitanus} across different intermediate CO$_2$ concentrations.
We also compared the growth phenotypes of partial CCM constructs in CO$_2$ concentration dependent strains of \textit{E. coli} and \textit{C. necator} that we constructed.
We found that introduction of an uncoupled CA (non-transporter) or a coupled CA (C$_i$ transporter) provided a fitness benefit at intermediate CO$_2$ concentrations.
We also found that while combining either of these activities with the $\alpha$-carboxysome on their own provided no benefit, combining them with each other provided benefits in some situations. 
This was unexpected, because including a coupled and an uncoupled CA in the same cell without encapsulation is expected to produce a futile cycle.
Our mathematical modeling suggests that at intermediate CO$_2$ concentrations and high growth rates the cell can become limited by both CO$_2$ and HCO$_{3}^{-}$ and C$_i$ consumption is high enough that cycling is actually beneficial for providing both C$_i$ species for growth.
This revealed a possible path to evolving a CCM through only fitness positive steps by acquiring first either a coupled or uncoupled CA, then acquiring the other type of CA as CO$_2$ fall further, and finally evolving an $\alpha$-carboxysome as CO$_2$ concentrations approach modern levels.
%
\subsubsection{Influence/Application}
This work provides insight into the evolution of the CCM and into possible life strategies of modern organisms living in high CO$_2$ environments.
These strategies might also be useful for improving the growth of industrial autotrophs at intermediate CO$_2$ concentrations.
Further, showing the functional expression of the DAB in \textit{C. necator} offers the potential for using the DAB to improve bioplastics production.
%
\subsubsection{My role}
My role in this project was to measure the effect of all CCM gene knockouts on the growth of \textit{H. neapolitanus} across a panel of intermediate CO$_2$ concentrations to establish which components of it's CCM are needed for growth at intermediate CO$_2$ concentrations.
%
\nocite{Flamholz2022-yo}
\printbibliography[heading=none]

% \leavevmode\newline
\leavevmode\pagebreak


\newrefsection
\subsection{Development of nuclease amplification for cas13 viral diagnostics:}
\subsubsection{Historical background}
CRISPR based diagnostics offer an attractive option for rapid point of care or at home detection of nucleic acids, such as virus genomes.
Their advantages include fast detection times, ease of conversion into either lateral flow or fluorescence assays, and simple operation (they do not require multiple liquid handling or incubation steps).
However existing CRISPR diagnostics were not sensitive enough to detect clinically relevant levels of SARS-COV2 in patient samples without pre-amplification of target sequences which negated all of these advantages.
Class III CRISPR systems include a cyclic-oligo-A activated nuclease csm6 that is used to amplify the CRISPR immune response.
We were interested in determining if we could link cas13 detection of viral RNAs to csm6 activation to improve limit of detection and allow SARS-COV2 detection in clinical samples.
%
\subsubsection{Central finding}
We found that cleavage of an A4-U6 substrate by cas13 produced a good activator for csm6.
However, the reaction appeared to be self limiting.
Kinetic modeling and reagent spike in experiments suggested that csm6's intrinsic activator cleavage activity was causing the self-limitation.
By using a single-fluoro modified activator, we were able to remove this self limitation and detect SARS-COV2 genomes in clinical samples.
We also developed a microfluidic device for automating sample processing and assay performance.
%
\subsubsection{Influence/Application}
This work has demonstrated a new method of improving time to detection and sensitivity in CRISPR diagnostics. 
This work will contribute to new and improved diagnostics technologies for the detection of a variety of clinically and scientifically relevant nucleic acids.
%
\subsubsection{My role}
My role in this work was to perform kinetic modeling of different potential reaction setups to predict potential improvement in time to detection or limit of detection, this included the analysis that suggested that using an uncleavable activator would remove self-limitation.
I also wrote analysis pipelines to process data and perform statistical analysis of data, including developing methods for detecting positive samples from microfluidic device data.
%
\nocite{Liu2021-pu}
\printbibliography[heading=none]

% \leavevmode\newline
\leavevmode\pagebreak


\newrefsection
\subsection{Discovery of CasX:}
\subsubsection{Historical background}
The field of genome editing was launched by the discovery of RNA guided DNA cleaving nucleases.
At the time there were only two such families, cas9 and cas12a, and there was interest in finding new programmable DNA nucleases with smaller sizes that would make them easier to deliver in therapeutics and potentially new mechanisms.
CasX was discovered as a <1,000 Amino acids RuvC containing protein in CRISPR loci and shown to be capable of protecting from plasmid transformation.
%
\subsubsection{Central finding}
We demonstrated that CasX was capable of cleaving dsDNA in cis and ssDNA in trans when provided with a complementary sgRNA \textit{in vitro}.
We showed induction of NHEJ at the targeted locus in human HEK293 T cells with wild type CasX and CRISPRi knockdown of genes in \textit{E. coli} with dead casX.
We solved cryo-EM to solve the structure of dcasX in complex with DNA and showed that there were two new domains the non-target strand binding (NTSB) domain and the target strand loading (TSL) domain.
We showed that the NTSB is needed for unwinding dsDNA.
We also detected a putative zinc binding motif in the TSL and showed that casX purifies bound to zinc.
%
\subsubsection{Influence/Application}
CasX provides a new modality for genome editing that is proving useful in a variety of applications and provides hope for aiding the treatment of a wide variety of diseases.
Scribe Therapeutics is pursuing casX based therapies.
%
\subsubsection{My role}
My role in this project was to measure zinc content in purified protein using x-ray fluorescence spectroscopy.
%
\nocite{Liu2019-nk}
\printbibliography[heading=none]



\end{document}

