%!TEX output_directory = build
%!TEX copy_output_on_build = true

\documentclass{article}

\raggedright

\usepackage[style=authoryear, 
            backend=biber,
            url=false,
            eprint=false, 
            dashed=false,
            maxnames=999]{biblatex}
\addbibresource{./mypapers.bib} %Imports bibliography file

\newcounter{namesnotimportant}
\newtoggle{ellipsis}

\makeatletter
\newbibmacro*{name:etal:delim}[1]{%
  \ifnumgreater{\value{listcount}}{\value{liststart}}
    {\ifboolexpr{
       test {\ifnumless{\value{listcount}}{\value{liststop}}}
       or
       test \ifmorenames
       or test {\ifnumcomp{\value{namesnotimportant}}{>}{0}}
     }
       {\printdelim{multinamedelim}}
       {\lbx@finalnamedelim{#1}}}
    {}}
\makeatother

\DeclareNameFormat{given-family-etal}{%
  \letbibmacro{name:delim}{name:etal:delim}%
  \ifnumcomp{\value{listcount}}{=}{1}
    {\setcounter{namesnotimportant}{0}%
     \global\toggletrue{ellipsis}}
    {}%
  \ifboolexpr{test {\ifnumcomp{\value{listcount}}{=}{1}}
              or test {\ifnumcomp{\value{listtotal}}{=}{2}}}
    {\ifgiveninits
      {\usebibmacro{name:given-family}
         {\namepartfamily}
         {\namepartgiveni}
         {\namepartprefix}
         {\namepartsuffix}}
      {\usebibmacro{name:given-family}
         {\namepartfamily}
         {\namepartgiven}
         {\namepartprefix}
         {\namepartsuffix}}}%
    {\ifboolexpr{test {\iffieldequalstr{hash}{d044631f3f9582c1e40e11481a6d06e9}}
            or 
            test {\iffieldequalstr{hash}{d36f79db286d144cbc39cc5559a702d3}}
            or 
            test {\iffieldequalstr{hash}{02913108490bef3778a320768a68b0d9}}}%% <----- put the correct hash here
      {\global\toggletrue{ellipsis}%
       \ifgiveninits
        {\usebibmacro{name:given-family}
           {\namepartfamily}
           {\namepartgiveni}
           {\namepartprefix}
           {\namepartsuffix}}
        {\usebibmacro{name:given-family}
           {\namepartfamily}
           {\namepartgiven}
           {\namepartprefix}
           {\namepartsuffix}}}%
      {\stepcounter{namesnotimportant}%
       \iftoggle{ellipsis}
         {\addcomma\space\textellipsis\global\togglefalse{ellipsis}\isdot}
         {}}}%
  \ifboolexpr{
    test {\ifnumequal{\value{listcount}}{\value{liststop}}}
    and
    (test \ifmorenames
     or test {\ifnumcomp{\value{namesnotimportant}}{>}{0}})
  }
    {\andothersdelim\bibstring{andothers}}
    {}}

\DeclareNameAlias{sortname}{given-family-etal}
\DeclareNameAlias{author}{given-family-etal}
\DeclareNameAlias{editor}{given-family-etal}
\DeclareNameAlias{translator}{given-family-etal}


\usepackage{soul}

\usepackage[letterpaper,
            bindingoffset=0.2in,
            left=1in,
            right=1in,
            top=1in,
            bottom=1in,
            footskip=.25in]{geometry}

\usepackage{titlesec} 
\titleformat{\section}[runin]
  {\normalfont\normalsize\bfseries}{}{0pt}{}
\titlespacing{\section}{0pt}{-\parskip}{1mm}

\renewcommand{\thesubsection}{\arabic{subsection}}
\titleformat{\subsection}[runin]
  {\normalfont\normalsize\bfseries}{Contribution \thesubsection: }{0pt}{}
\titlespacing{\subsection}{0pt}{-\parskip}{1mm}

\titleformat{\subsubsection}[runin]
  {\normalfont\normalsize\itshape}{\thesubsubsection: }{0pt}{\ul}
\titlespacing{\subsubsection}{0pt}{-\parskip}{1mm}

\begin{document}


\section*{Contributions to Science:}
\newrefsection
\subsection{A new type of inorganic carbon pump that drives CO$_2$ concentration:}
\subsubsection{Historical background}
The enzyme rubisco fixes $>$99.5\% of the CO$_2$ entering the biosphere each year and is essential in plants, algae, and most autotrophic bacteria. 
However, Rubisco is inhibited by O$_2$, a problem in the the modern atmosphere with its 20\% O$_2$ and only 0.04\% CO$_2$.
Many bacteria overcome this using an $\alpha$-carboxysome based CO$_2$ concentrating mechanism ($\alpha$-CCM).
These systems rely on HCO$_{3}^{-}$ pumping, however, the mechanism of HCO$_{3}^{-}$ pumping was unknown in chemotrophs.
%
\subsubsection{Central finding}
To identify HCO$_{3}^{-}$ pumps, I screened for $\alpha$-CCM genes in the model chemotroph \textit{H. neapolitanus}.
I identified two putative transporter operons, then showed sufficiency for pumping in \textit{E. coli}. 
Unexpectedly, the data were consistent with energy coupled carbonic anhydrase (CA) activity not direct pumping.
This causes HCO$_{3}^{-}$ flux by converting membrane permeable CO$_2$ into membrane impermeable HCO$_{3}^{-}$ trapping it in the cell.
I showed homologs in the pathogens \textit{V. Cholera} and \textit{B. anthracis} had the same activity.
%
\subsubsection{My role}
I conceived and designed the experiments, performed the genetic screens, analyzed sequencing data, performed the mechanistic experiments, and purified protein.
%
\subsubsection{Influence/Application}
This work identified a new family of energy coupled CAs, only the second such family known.
This work enabled reconstitution of a functional $\alpha$-CCM in \textit{E. coli}. 
A homolog found in \textit{S. aureus} has the same function and is essential for growth in air. 
Factoring in energy coupled CAs aided interpretation of carbon isotope fractionation in rock strata.
There are proposed applications of these pumps in engineering crop plants and autotrophic bio-fuel production hosts.
%
\nocite{Desmarais2019-yc,Desmarais2018-ac,Desmarais2019-dc}
\printbibliography[heading=none]

% \leavevmode\newline
% \leavevmode\pagebreak

\newrefsection
\subsection{Potential evolutionary paths of carbon dioxide concentrating mechanisms:}\label{CCM_EVO}
\subsubsection{Historical background}
The $\alpha$-carboxysome based CO$_2$ concentrating mechanism ($\alpha$-CCM) required several major evolutionary steps to evolve.
However, none of the potential intermediates are expected to provide a fitness benefit in modern conditions so it is not clear how it evolved.
The atmosphere was very different when the $\alpha$-CCM evolved, with much higher levels of CO$_2$ and much lower levels of O$_2$.
This lead us to hypothesize that evolutionary intermediates of the CCM may have provided fitness benefits at intermediate atmospheric compositions.
%
\subsubsection{Central finding}
Evolving an $\alpha$-CCM required acquiring a CA, gaining a HCO$_{3}^{-}$ pump, and co-encapsulating CA and rubisco. 
Removing any of these stops $\alpha$-CCM function in normal atmosphere.
We measured the effect of $\alpha$-CCM gene knockouts in \textit{H. neapolitanus} across different CO$_2$ concentrations.
We also measured  the CO$_2$ dependent phenotypes of potential evolutionary intermediates in reporter strains of \textit{E. coli} and \textit{C. necator} that we constructed.
We modeled carbon fluxes as a function of growth rate and CO$_2$ concentration.
This data suggested that as CO$_2$ concentrations fall, HCO$_{3}^{-}$ becomes limiting before CO$_2$.
This suggested that as CO$_2$ started to fall either a pump or CA can help.
As levels fall further CO$_2$ and HCO$_{3}^{-}$ become co-limiting and having both a CA and a pump provides a benefit despite the potential for producing a futile cycle
Eventually, only a full $\alpha$-CCM will work.
This provides a potential path for the evolution of a $\alpha$-CCM.
%
\subsubsection{My role}
I performed a massively parallel growth assay of gene knockouts in \textit{H. neapolitanus} across intermediate CO$_2$ concentrations to identify CCM genes needed at intermediate CO$_2$ concentrations.
%
\subsubsection{Influence/Application}
This work provides insight into the evolution of the $\alpha$-CCM and into possible life strategies of modern organisms living in high CO$_2$ environments.
These advances might also be useful for improving the growth of industrial autotrophs.
Further, showing expression of the pumps in \textit{C. necator} offers the potential to improve bio-plastics production.

%
\nocite{Flamholz2022-yo}
\printbibliography[heading=none]

% \leavevmode\newline
% \leavevmode\pagebreak

% \newrefsection
\subsection{General epistasis protein fitness landscape mapping and design:}
\subsubsection{Historical background}
Mutational scanning maps a protein's fitness landscape by measuring the fitness of all single mutants.
This information is used for variant effect prediction and design.
However, mutational scanning experiments require production of all single mutants and only provide information on fitness in the protein's local context, limiting their utility for divergent proteins.
Being able to learn from data sets that are easier to generate and provide information over a wider area would be greatly beneficial.
Current efforts to learn from random mutagenesis have relied on neural networks or linear methods.
However, linear methods miss nonlinearities in the data's true structure and neural networks cannot be inspected to gain insight into function.
General epistatic models capture nonlinear genotype-phenotype relationships without sacrificing interpretability but have not been applied to protein design.
Phylogentic data also provide insight into fitness landscapes but inspectable models that are able to leverage both phylogenetic and experimental data have been rare.
%
\subsubsection{Central finding}
We performed random mutagenesis on dihydrofolate reductase and measured enzyme activity in a massively parallel growth assay. 
I trained models including linear models, general epistatic models, and large neural nets with and without access to phylogenetic data. 
I showed that the general epistatic models had simmilar performance to the neural net on held out test data.
Both these models showed improvement when provided with phylogenetic data but out performed the phylogenetic data alone or with the linear model. 
I also used each model to design new variants while systematically varying both number of mutations and optimization strategy to evaluate how well each model is able to extrapolate.
I used a massively parallel growth assay to validate 12,000 designed proteins.
This work is currently awaiting sequencing results from the final experiment before submission.
%
\subsubsection{Influence/Application}
This work provides new methods for mapping the genotype-phenotype landscape of proteins and designing new variants.
It also provides simple directly inspectable models that produce performance on par with neural net approaches for some prediction and design tasks.
%
\subsubsection{My role}
I am the primary author on this work.
I designed and conceived of the experiments, produced mutant libraries, performed massively parallel growth assays, wrote analysis code, wrote model code, trained models, tested model performance, and evaluated optimized sequence behavior.
%
% \printbibliography[heading=none]

% \leavevmode\newline
% \leavevmode\pagebreak


\newrefsection
\subsection{Development of new CRISPR tools:}
\subsubsection{Historical background}
The ever expanding suite of CRISPR tools has helped drive a biotech revolution.
I have been part of two tool development projects, identifying CasX as a new RNA guided DNA nuclease and the development of Cas13/Csm6 RNA diagnostics.
%
\subsubsection{CasX genome editing}
RNA guided DNA nucleases launched the CRISPR field.
However, nuclease size has hindered therapeutic applications.
CasX is a $<$1,000 amino acid RuvC containing protein from CRISPR loci.
We demonstrated guide directed cutting activity \textit{in vitro} as well as cutting and CRISPRi \textit{in vivo}.
We solved a structure of the complex with target DNA and identified two new domains.
We also detected a zinc binding motif and showed that CasX binds zinc.
My role was to use x-ray fluorescence to detect zinc bound to the purified protein.
CasX provides a new modality for genome editing that is proving useful in a variety of applications.
Scribe Therapeutics is pursuing casX based therapies.
%
\subsubsection{Csm6 boosted Cas13 RNA detection}
CRISPR diagnostics can detect nucleic acids in one-pot isothermal reactions.
This makes them attractive for at-home or point-of-care diagnostics.
However, poor sensitivity meant pre-amplification was required to detect SARS-COV2 in patient samples.
Class III CRISPR systems include a cyclic-oligo-A activated nuclease, Csm6.
We hypothesized that linking cas13 and csm6 using cas13 targets that release Csm6 activator upon cleavage would improve sensitivity.
We used kinetic modeling and spike in assays to show that secondary activator cleavage was limiting sensitivity.
Using an activator resistant to secondary cleavage allowed detection of SARS-COV2 in clinical samples.
My role in this work was kinetic modeling and writing the data analysis and statistical pipelines.
This work has improved time to detection and sensitivity in CRISPR diagnostics.
%

\nocite{Liu2021-pu,Liu2019-nk}
\printbibliography[heading=none]

% \leavevmode\newline
% \leavevmode\pagebreak


\newrefsection
\subsection{Driving carbon flux toward chemical production by engineering glucose uptake during nitrogen starvation:}
\subsubsection{Historical background}
Using microbial hosts to produce chemicals offers the potential to produce a variety of compounds from renewable feed-stock.
However, production hosts frequently loose production efficiency as natural selection drives evolution towards redirecting carbon and energy flux towards growth not production.
This can be overcome by coupling production of the desired chemical to cell fitness, but in many cases this is not possible.
An alternative strategy is growth decoupling, in which production hosts are grown up, then growth is stopped and all metabolic flux is directed towards production.
However, when growth is stopped, many chassis organisms including \textit{E. coli} slow and eventually halt their metabolism stopping production.
A general strategy for enhancing metabolic rate during growth decoupling would dramatically improve prospects for engineered chemical production in biological hosts.
%
\subsubsection{Central finding}
We found that by over-expressing PstI we were able to increase glucose uptake after growth was stopped by nitrogen limitation. 
However we did not find this increased glucose consumption increased yield significantly, and it is likely additional work will be needed to direct this increased flux towards chemical production.
%
\subsubsection{Influence/Application}
This work provides another tool that metabolic engineers can use to optimize the production of their compound of interest.
%
\subsubsection{My role}
My role in this project was to perform growth and chemical production assays with modified strains and to prepare samples for mass spectrometry.
%
\nocite{Chubukov2017-uu}
\printbibliography[heading=none]

% \leavevmode\newline



\end{document}

