%!TEX output_directory = build
%!TEX copy_output_on_build = true

\documentclass{article}

\raggedright

\usepackage[style=numeric-comp, backend=biber,url=false,eprint=false]{biblatex}
\addbibresource{./mypapers.bib} %Imports bibliography file

\usepackage{soul}

\usepackage[letterpaper,
            bindingoffset=0.2in,
            left=1in,
            right=1in,
            top=1in,
            bottom=1in,
            footskip=.25in]{geometry}

\usepackage{titlesec} 
\titleformat{\section}[runin]
  {\normalfont\normalsize\bfseries}{}{0pt}{}
\titlespacing{\section}{0pt}{-\parskip}{1mm}

\renewcommand{\thesubsection}{\arabic{subsection}}
\titleformat{\subsection}[runin]
  {\normalfont\normalsize\bfseries}{Contribution \thesubsection: }{0pt}{}
\titlespacing{\subsection}{0pt}{-\parskip}{1mm}

\titleformat{\subsubsection}[runin]
  {\normalfont\normalsize\itshape}{\thesubsubsection: }{0pt}{\ul}
\titlespacing{\subsubsection}{0pt}{-\parskip}{1mm}

% \subsection{Driving carbon flux toward chemical production by engineering glucose uptake during nitrogen starvation:}
% \subsubsection{Historical background}

\begin{document}
\section*{Personal statement:}
I am uniquely qualified for this F32 application because it leverages my expertise in massively parallel assays, biochemistry and, quantitative modeling.
\subsubsection*{massively parallel assays}
In graduate school, I performed a massively parallel growth assay to measure quantitative phenotypes for $\sim$100,000 non-lethal (at 10\% CO$_2$) gene knockouts in the chemolithoautotroph \textit{H. neapolitanus} as a function of CO$_2$ concentration.\supercite{Desmarais2019-yc,Flamholz2022-yo}
After this, I used massively parallel growth assays to measure enzyme kinetics \textit{in vivo} by calibrating growth rates of individual strains in mixed cultures kept at mid log phase against measured enzyme kinetic parameters in work I hope will be published soon.
Through this experience, I learned how to design, perform, optimize, and analyze massively parallel assays to gather high quality quantitative data that varies as a function of external variables.
I also have experience with building mutant libraries using programed transposon mutagenesis,\supercite{Desmarais2019-yc} error prone PCR, and mutated oligo libraries as well as preparing libraries for Illumina sequencing.\supercite{Desmarais2019-yc}
This experience will aid me in designing and performing the massively parallel splicing assays I propose in this work.
\subsubsection*{biochemistry} %I think this may be the wrong term here, I want to also include my uptake/reporter assays... think this over
I have experience using biochemisty(?) techniques to follow up on interesting hits from screens. 
This includes techniques as diverse as reporter strain growth assays, C$^14$ uptake assays, protein co-purification purification, enzyme assays, site specific mutagenesis, and x-ray fluorescence spectroscopy.\supercite{Desmarais2019-yc,Liu2019-nk} 
This experience will prepare me for the work I have proposed here to investigate the mechanisms of interesting hits from the screen.
\subsubsection*{quantitative modeling}
I have experience writing data analysis pipelines for Illumina sequencing data and for quantitative data more broadly.\supercite{Desmarais2019-yc,Liu2021-pu,Flamholz2022-yo}
Further I have demonstrated expertise performing kinetic modeling of complicated multi-step reactions.\supercite{Liu2021-pu}
Finally, in recent as yet unpublished work, I have developed skill at modeling with machine learning models including neural nets, general epistasis models, and potts models.
These skills will prepare me well for building quantitative models of splicing informed by both phylogeny and my massively parallel assay data as I propose here.



\printbibliography[heading=none]


\end{document}